Blish HUD/Common/Gw2/Chat.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Blish_HUD.Controls.Extern;
+using Blish_HUD.Controls.Intern;
+using System.Linq;
+
+namespace Blish_HUD.Common.Gw2 {
+
+    /// <summary>
+    /// Methods related to interaction with the in-game chat.
+    /// </summary>
+    public static class Chat {
```

Comment:
I think this would end up being better exposed under the GameIntegration service.  Something like `GameService.GameIntegration.Chat`.  Especially since it already depends on so much from it.

Reply:
Noted.

Reply:
Review: https://github.com/blish-hud/Blish-HUD/pull/232/commits/f9ce5a737eec4052bf3b0d0e6b5af9962a9578f8